commands/web/xb-ui-dev

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+## #ddev-generated
+## Description: Install dependencies and run Experience Builder's UI app development server.
+## Example: ddev xb-ui-dev
+## Aliases: ui-dev,dev
+
+cd "$DDEV_COMPOSER_ROOT/$DDEV_DOCROOT/modules/contrib/experience_builder/ui" || exit 1
+
+npm install
+npm run drupaldev -- --host
```

Comment:
So this is something you would execute and then leave running in the terminal while you develop, is that right? If so, we can probably run it as a service in the container so it's always available without the user having to manually run a command at all. (I don't know off the top of my head how you add a background service like that--only that I know it can be done because I saw it somewhere.)

Reply:
Ah, I see. I didn't realize this was mutually exclusive with `npm run build`. That clarifies things. So your reasoning seems good to me. Tell me: as a target user, would you rather have two separate commands, i.e., `xb-ui-build` and `xb-ui-dev`? or one command with options, e.g., `xb-ui-build` and `xb-ui-build --dev` or something? Either way is just as easy, so you tell me which way you would like it to be in daily use. Note: If we keep them as separate commands, the new one will have to be added to `project_files` in https://github.com/TravisCarden/ddev-drupal-xb-dev/blob/v0.0.16/install.yaml#L7.

Reply:
Another really thoughtful question! The convention is to keep them separate. (A few examples: [Vite](https://vite.dev/guide/cli.html), [Create React App](https://create-react-app.dev/docs/getting-started#scripts), [Next.js CLI](https://nextjs.org/docs/app/api-reference/cli/next), [`create-preact`](https://github.com/preactjs/create-preact/blob/0.5.0/templates/base/package.json), [Astro's CLI](https://docs.astro.build/en/reference/cli-reference/).)

Thanks for pointing out that I missed adding the new command under `project_files`. Fixed in [`e5df5`](https://github.com/TravisCarden/ddev-drupal-xb-dev/pull/19/commits/e5df57558a4877b8926687c5ada3bd5b14d5ff00).

Reply:
Oh, great! It always makes me happy when there's a clear convention to follow.